setup.py

```diff
@@ -52,7 +52,7 @@ def run_tests(self):
     ],
     keywords='gds govuk notify',
 
-    packages=find_packages(),
+    packages=find_packages(include=['notifications_python_client']),
```

Comment:
There's only one root package so it's easier to include it (and potential future children) than exclude the other 3 top level modules